### DISCUSSION TITLE
Is it possible to deploy Zitadel using kustomize without  ORBOS?

### DISCUSSION THREAD

**Discussion by number5** [zitadel/zitadel#1406] [Q&A] [answered]
I can see that you have a kustomization folder at `k8s/base`,  so is it possible to deploy Zitadel using kustomize without  ORBOS?

**Comment by stebenz** [comment, marked as answer]
Yes, Zitadel can be run without ORBOS, but I would recommend using the zitadelctl, which is attached to each release, to deploy the database-operator and the Zitadel-operator together. The operators are self-reconciling as long as a git repository with the required YAML-files is available. 
These files in the kustomize-folder are unfortunately relics from before we implemented the operator for Zitadel, it is theoretically still possible to deploy it with kustomize but not currently supported by us.
Due to complexity, we moved to an operator approach which covered tasks like DNS records, firewall, database maintaining, IAM configuration, reconciling, data migrations, IAM setup, and other functionalities for our needs. 
We are currently using the techstack cloudflare -> orbos cluster -> ambassador -> Zitadel -> cockroach, other tech stack are currently not supported. In addition to that, we are working with the gitops-pattern, meaning you need a git repository to start using the zitadelctl.

If you have any specific questions don't hesitate to ask.

**Reply by thesephirot** [reply]
Hi Bruce,

thank you for your interest in our product. 

For a quick check of the feature set we provide I would recommend to try http://zitadel.ch/ and register a free tier organisation.   

see https://docs.zitadel.ch/start/

In terms of installation, as Stefan mentioned: It is definitely possible to run Zitadel based on kustomization files. (thats how we did it in the past). Due to complexity we switched to an operator to ease things out. The documentation for the operator as well as example files will follow. This is WIP and will follow soon.

Regards,

Christian

**Reply by number5** [reply]
Thanks @stebenz and @thesephirot, will try out `zitadelctl` when I have a chance.

**Reply by thesephirot** [reply]
@number5 fine!, we`d love to hear your feedback. At the moment zitadelctl is capable to fulfill gitops deployments with a dedicated ops repository it reads its configurations from. we are currently working on a crd mode that enables customers to use it without the need of a separate dependency such as git. this will happen shortly. Greetings to downunder :)
